HoneyDo.Web/ClientApp/package.json

```diff
@@ -35,8 +29,10 @@
     "test": "react-scripts test --env=jsdom",
     "test:ci": "CI=true react-scripts test",
     "eject": "react-scripts eject",
-    "styleguide": "styleguidist server",
-    "styleguide:build": "styleguidist build"
+    "lint": "eslint src"
+  },
+  "eslintConfig": {
```

Comment:
This will need to move to `eslintrc.json` or `eslint.json` needs to be removed (I confirmed, you can have it in either place, I'd stick with the package.json).